spec/features/user_delete_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
```

Comment:
Do the load order changes mean that all of our spec files should require `spec_helper` now?